From 0e41b11b23c91293d1b39a8ec4cb80c68fb26ad7 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 28 Apr 2022 07:35:31 -0700
Subject: [PATCH] meson: Do not run git rev-parse during configure

This option will try to deduce if babl is being built from git tree or
release tarball, there should be a better way like checking for .git
directory etc. instead of doing git operations needing network

see
https://gitlab.gnome.org/GNOME/babl/-/commit/2dc7fc403fe427a889913ef0cfb71de85b4326ec#note_1439732

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 7e7a935..649b456 100644
--- a/meson.build
+++ b/meson.build
@@ -451,7 +451,7 @@ if git_bin.found() and run_command(
     git_bin,
     'rev-parse',
     '--is-inside-work-tree',
-    check: true,
+    check: false,
 ).returncode() == 0
   git_version_h = vcs_tag(
     input : 'git-version.h.in',
-- 
2.36.0

